packages/toolpad-components/src/Text.tsx

```diff
@@ -23,9 +22,18 @@ interface TextProps extends Omit<BaseProps, 'children'> {
 
 const MarkdownContainer = styled('div')({
   display: 'block',
-  [`&:empty::before`]: { content: '""', display: 'inline-block' },
+  minHeight: '25px',
```

Comment:
I'd prefer this to be solved using a similar trick as the `&:empty::before` so that it follows `fontSize` rather than an absolute height